tools/ci/compile.sh

```diff
@@ -69,34 +80,38 @@ fi
 
 echo "============ Checking Javadocs ============"
 
+javadoc_output=/tmp/javadoc.out
+
 # use the same invocation as .github/workflows/docs.sh
-run_mvn javadoc:aggregate -DadditionalJOption='-Xdoclint:none' \
+$MVN javadoc:aggregate -DadditionalJOption='-Xdoclint:none' \
```

Comment:
```suggestion
$MVN javadoc:aggregate -T1 -DadditionalJOption='-Xdoclint:none' \
```
according to your statement "every call (apart from mvn exec has to run with a parallelism of 1.". But tbh, I don't see value here because we're not parsing the output. In this sense, it would be up for the user to decide. :shrug:

Reply:
javadoc can run in parallel.